src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/Mocks/Repository/InstanceEventsRepositoryMock.cs

```diff
@@ -27,47 +28,32 @@ public Task<InstanceEvent> InsertInstanceEvent(InstanceEvent instanceEvent)
             return Task.FromResult(instanceEvent);
         }
 
-        public Task<List<InstanceEvent>> ListInstanceEvents(string instanceId, string[] eventTypes, DateTime? fromDateTime, DateTime? toDateTime)
+        public async Task<List<InstanceEvent>> ListInstanceEvents(string instanceId, string[] eventTypes, DateTime? fromDateTime, DateTime? toDateTime)
         {
+            List<InstanceEvent> events = new List<InstanceEvent>();
+
             lock (TestDataUtil.dataLock)
```

Comment:
Is the lock needed? Seems like the code is only reading?

Reply:
I'm not sure. I'm would like to leave it in for now. The internals of `File.ReadAllText` doesn't open up for shared reads (`FileShare.Read`). Default behavior of `StreamReader` is to apply a read lock.